metadata.json

```diff
@@ -10,7 +10,10 @@
   "issues": "https://github.com/voxpupuli/puppet-nscd/issues",
   "tags": ["nscd", "dns", "dnscache", "cache" ],
   "dependencies": [ 
-    { "name": "puppetlabs/stdlib", "version_requirement": ">=2.2.1 < 5" }
+    {
+      "name": "puppetlabs-stdlib",
+      "version_requirement": ">= 4.6.0 < 5.0.0"
```

Comment:
If we keep this, we should use 4.13.1. This is the first really useful version for puppet4, as it ships the datatypes. They aren't used here, but we use 4.13.1 in 99% of our modules